bears/c_languages/GNUIndentBear.py

```diff
@@ -1,5 +1,5 @@
 import platform
-import shlex
+from shlex import split
```

Comment:
please keep the old one, we need to make some discussion out of this anyway, but as split is a builtin for string, it's better understandable together with the module name
```
"a b".split()
# vs
shlex.split("a b")
```